### DISCUSSION TITLE
ThunderHub Plugin

### DISCUSSION THREAD

**Discussion by raduButucelea23** [btcpayserver/btcpayserver#4389] [Plugins & integrations]
This thread is tracking the addition of ThunderHub as a plugin to BTCPay Server giving users a "one-click" installation option via the "Manage Plugins" menu.

**Comment by ndeet** [comment]
Hi you can do that on the docker install by adding a fragment, see [here](https://docs.btcpayserver.org/FAQ/LightningNetwork/#how-to-install-thunderhub). A plugin would only be built using C# on top of the BTCPay api. This is more a complete separate project that gets added via it's own container.

**Comment by Kukks** [comment]
ThunderHub has been supported for a while already.
